src/earn/EarnEnterAmount.tsx

```diff
@@ -100,6 +101,9 @@ function EarnEnterAmount({ route }: Props) {
   const isWithdrawal = mode === 'withdraw'
   const { depositToken, withdrawToken, eligibleSwappableTokens } = useTokens({ pool })
 
+  const depositStatus = useSelector(depositStatusSelector)
+  const transactionSubmitted = depositStatus === 'loading'
```

Comment:
think this screen is reused for withdraw too, should there be an additional check? probably not required, unless the redux state is corrupted

Reply:
I don't think this is strictly necessary, since pressing "continue" for a withdrawal actually takes you to another confirmation screen; I can leave a comment however.